### DISCUSSION TITLE
enquiry

### DISCUSSION THREAD

**Discussion by jesutobi** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#18] [Outreachy 2022] [answered]
Goodmorning @BatoolMM i would like to know what happens after i create an issue, should i go on to fix the issue or wait for a go ahead

**Comment by BatoolMM** [comment, marked as answer]
Hi @jesutobi, after you create an issue, please open a PR to address it - you don't need for feedback for the issue!

**Reply by jesutobi** [reply]
@BatoolMM i started working on the main branch before realising that i had to create a new branch...would the changes i have made on the main b
ranch reflect in the new branch i would create
